systemverilog-plugin/UhdmAst.cc

```diff
@@ -3713,17 +3713,31 @@ void UhdmAst::process_tagged_pattern()
     AST::AstNode *lhs_node = nullptr;
     if (assign_node) {
         assign_type = assign_node->type;
-        lhs_node = assign_node->children[0];
+        lhs_node = assign_node->children[0]->clone();
     } else {
         lhs_node = new AST::AstNode(AST::AST_IDENTIFIER);
         auto ancestor = find_ancestor({AST::AST_WIRE, AST::AST_MEMORY, AST::AST_PARAMETER, AST::AST_LOCALPARAM});
         if (!ancestor) {
+            // special case: if this assignment pattern is inside cell high conn port
+            if (find_ancestor({AST::AST_CELL})) {
+                auto typespec_h = vpi_handle(vpiTypespec, obj_h);
+                if (typespec_h && vpi_get(vpiType, typespec_h) == vpiStringTypespec) {
+                    // and we are setting all values to some constant
```

Comment:
```suggestion
                // and we are setting all values to some constant
                if (typespec_h && vpi_get(vpiType, typespec_h) == vpiStringTypespec) {
```
For consistency with the comment + if above